eng/liveBuilds.targets

```diff
@@ -260,10 +260,4 @@
           DependsOnTargets="
             ResolveLibrariesRefAssembliesFromLocalBuild;
             ResolveLibrariesRuntimeFilesFromLocalBuild" />
-
-  <PropertyGroup>
-    <!-- Keep in sync with outputs defined in Microsoft.NETCore.Platforms.csproj. -->
-    <BundledRuntimeIdentifierGraphFile>$([MSBuild]::NormalizePath('$(ArtifactsBinDir)', 'Microsoft.NETCore.Platforms', 'runtime.json'))</BundledRuntimeIdentifierGraphFile>
-    <BundledRuntimeIdentifierGraphFile Condition="!Exists('$(BundledRuntimeIdentifierGraphFile)')">$([MSBuild]::NormalizePath('$(LibrariesProjectRoot)', 'Microsoft.NETCore.Platforms', 'src', 'runtime.json'))</BundledRuntimeIdentifierGraphFile>
-  </PropertyGroup>
```

Comment:
@elinor-fung @ViktorHofer 

We've added these lines so the build SDK would know how the rid being built fits in the graph.
I'd be surprised if we can just remove them, as that would mean they were not needed.

Reply:
Yes, the SDK that gets built has the non-portable rid added to it.

That's not what these lines are for: they are for making the SDK that is during the build (for example: a portable Microsoft SDK) know the non-portable rid.